src/data/providers.ts

```diff
@@ -48,6 +48,14 @@ export const providersData: Providers = {
                 message: "Install Geyser in the Aternos plugin list and connect to your server with your Java IP and port. See [Aternos's article](https://support.aternos.org/hc/en-us/articles/360051047631) for more details. Do note: Aternos automatically installs the latest version of the plugin, but does not auto update it. To update Geyser on Aternos, simply reinstall the plugin (no need to uninstall the plugin/deleting the plugin files)."
             })
         },
+        {
+            name: 'AxentHost',
+            url: 'https://axenthost.com/',
+            description: translate({
+                id: 'providers.provider.axenthost.description',
+                message: "Once activated, navigate to the server Variants tab and select GeyserMC. Removing the server files in the Files tab beforehand will be required to make a clean installation of Geyser."
```

Comment:
Once what is activated? Further, which server files should/will be removed - does installing Geyser using this method remove all Geyser-related files, or will it create a fully fresh server? Please be specific; we're in the process of re-making our providers list and these specifics will matter :)

Reply:
The way it is made is you create a vanilla Minecraft server by default. Reinstalling the server doesn't delete the current files for no data loss, so it is recommended to delete the current server files before changing the variant.
I'll rewrite it, let me know if it matches what you expect !

Reply:
Hey @onebeastchris , any news about this ?